combat-trainer.lic

```diff
@@ -189,12 +191,46 @@ class LootProcess
   def dispose_body(game_state)
     return if DRRoom.dead_npcs.length < 1
     game_state['mob_died'] = true
-
-    check_skinning(DRRoom.dead_npcs.first) if @skin
-
+      if @necro
```

Comment:
also, I think the problem might be that I'm not calling in my variables correctly.

when I def main(combat_vars)
  setup

variablename = combat_vars.skinning['variable']

that defines variablename as whatever is in the yaml under skinning then looks at the left side if the colon for variable and then sets variablename as whatever is on the right side of the colon, correct?

example, if I had this method in my yaml,

ex1:
  exF: exA

then this def in combat-training,

variablename = combart_vars.ex1['exF']

it would set variablename to exA?